package.json

```diff
@@ -2,16 +2,39 @@
   "name": "minecraft-data",
   "version": "2.70.0",
   "description": "Provide easy access to minecraft data in node.js",
-  "main": "index.js",
-  "tonicExampleFilename": "example.js",
+  "main": "build/main/index.js",
+  "typings": "build/main/index.d.ts",
+  "module": "build/module/index.js",
+  "tonicExampleFilename": "./src/example.js",
   "scripts": {
-    "generate:types": "node typings/generate-typings.js",
-    "generate:data": "./bin/generate_data.js",
-    "test:types": "tsc typings/test-typings && node typings/test-typings.js",
-    "test": "require-self && npm run generate:data && npm run lint && npm run test:types && mocha",
-    "lint": "standard",
-    "prepublish": "require-self && npm run generate:data && npm run generate:types",
-    "fix": "standard --fix"
+    "build": "run-p build:*",
+    "build:main": "tsc -p tsconfig.json",
+    "build:module": "tsc -p tsconfig.module.json",
+    "fix": "run-s fix:*",
+    "fix:prettier": "prettier \"src/**/*.ts\" --write",
+    "fix:lint": "eslint src --ext .ts --fix",
+    "test": "run-s build test:*",
+    "test:lint": "eslint src --ext .ts",
+    "test:prettier": "prettier \"src/**/*.ts\" --list-different",
+    "test:spelling": "cspell \"{README.md,.github/*.md,src/**/*.ts}\"",
+    "test:unit": "nyc --silent ava",
+    "check-cli": "run-s test diff-integration-tests check-integration-tests",
+    "check-integration-tests": "run-s check-integration-test:*",
```

Comment:
This is way too much boilerplate

Reply:
Yep just had some base typescript conversions which I made and am just removing unnecessary features now